environment.yml

```diff
@@ -45,8 +46,8 @@ dependencies:
         - Cython
         - redis
         - keras==1.2.1
-        - git+https://github.com/Theano/Theano.git@adfe319ce6b781083d8dc3200fb4481b00853791#egg=Theano
-        - git+https://github.com/neocxi/Lasagne.git@484866cf8b38d878e92d521be445968531646bb8#egg=Lasagne
+        - git+https://github.com/Lasagne/Lasagne.git@7992faa80fa5233a786e2582a605e854cea7d1cf#egg=Lasagne
+        - git+https://github.com/Theano/Theano.git@2c19431b165aaf8289d1e6654988c64f4bb234e1#egg=Theano
```

Comment:
Theano 1.0 is available directly from pip:
![image](https://user-images.githubusercontent.com/35857569/40359788-c4a748f4-5d78-11e8-9e6e-04ebcc99ec82.png)
Putting this package looks cleaner to me:
- Theano==1.0

Reply:
Good catch. But shouldn't it be `Theano=1.0.1`?

Also, it is a bit confusing, because the Theano documentation "recommends" installing directly from git:

> Bleeding-Edge Installation (recommended)
> Install the latest, bleeding-edge, development version of Theano with:
> 
> <sudo> pip install <--user> <--no-deps> git+https://github.com/Theano/Theano.git#egg=Theano
> Any argument between <...> is optional.
> Use sudo for a root installation.
> Use user for a user installation without admin rights. It will install Theano in your local site-packages.
> Use no-deps when you don’t want the dependencies of Theano to be installed through pip. This is important when they have already been installed as system packages.
> If you encountered any trouble, head to the Troubleshooting page.

I think that is left over from when Theano was actively developed (it is not anymore), and when the developers didn't necessarily post all updates to pip.

Anyway, I like the pip approach better, but I'm not going to block merging on it. @cjcchen you can change it if you want, or just merge.